devise_security_extension.gemspec

```diff
@@ -5,7 +5,10 @@
 
 Gem::Specification.new do |s|
   s.name = "devise_security_extension"
-  s.version = "0.8.0"
+  lib = File.expand_path('../lib', __FILE__)
```

Comment:
I think we usually put these 3 lines at the beginning of gemspec file.

Reply:
I pulled the first 2 lines up

Reply:
lib should be undefined by the line 6

Reply:
oh yeah, I botched that edit